adapters/kargo/kargotest/exemplary/banner.json

```diff
@@ -0,0 +1,221 @@
+{
+  "mockBidRequest": {
+    "id": "5f4d1e01",
+    "at": 1,
+    "imp": [
+      {
+        "id": "8b1bdcca",
+        "banner": {
+          "w": 300,
+          "h": 300,
+          "format": [
+            {
+              "w": 300,
+              "h": 250
+            }
+          ],
+          "pos": 0,
+          "expdir": [1, 2, 3, 4, 5]
+        },
+        "pmp": {
+          "deals": [
+            {
+              "id": "TTD",
+              "ext": {
+              }
+            }
+          ]
+        },
+        "tagid": "73",
+        "secure": 1,
+        "iframebuster": ["ALL"],
+        "ext": {
+        }
```

Comment:
Exemplary files `adapters/kargo/kargotest/exemplary/banner.json`, `adapters/kargo/kargotest/exemplary/native.json` and `adapters/kargo/kargotest/exemplary/video.json` don't include the `"tagid"` field in their `request.imp[i].ext` fields:
```
 1 {
 2   "mockBidRequest": {
 3     "id": "5f4d1e01",
 4     "at": 1,
 5     "imp": [
 6       {
 7         "id": "8b1bdcca",
 8 *-- 21 lines: "banner": {------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
29         "tagid": "73",
30         "secure": 1,
31         "iframebuster": ["ALL"],
32         "ext": {
33         }
34       }
35     ],
adapters/kargo/kargotest/exemplary/banner.json
```

```
 1 {
 2     "mockBidRequest": {
 3       "id": "5f4d1e01",
 4       "at": 1,
 5       "imp": [
 6           {
 7               "id": "8b1bdcca",
 8 *-- 22 lines: "banner": {------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
30               "tagid": "73",
31               "secure": 1,
32               "iframebuster": [
33                   "ALL"
34               ],
35               "ext": {
36               }
37           }
38       ],
adapters/kargo/kargotest/exemplary/native.json
```

```
 1 {
 2     "mockBidRequest": {
 3       "id": "5f4d1e01",
 4       "at": 1,
 5       "imp": [
 6           {
 7               "id": "8b1bdcca",
 8 +-- 27 lines: "banner": {------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
35               "tagid": "73",
36               "secure": 1,
37               "iframebuster": [
38                   "ALL"
39               ],
40               "ext": {
41               }
42           }
43       ],
adapters/kargo/kargotest/exemplary/video.json
```
Given the fact that `"tagid"` was marked as `required` inside `static/bidder-params/kargo.json`, none of the mock bid requests inside the files above would have made it to your adapter: Prebid Server Core would have discarded them. Can we make them more realistic?

Reply:
This is probably not necessary in `req.imp[i].ext`, but we need either the `tagid`, `placementid` or `adSlotId`

Reply:
Maybe a check can be added inside `MakeRequests` on `req.imp[i].tagid`

Reply:
Yeah I can add a check there. 
I'm discussing it with my team and we might go with another param such as `AdSlotID` or `AdSlot` for `req.imp[i].ext` instead. I'll confirm everything and re-request a review when its ready.